docs-source/tutorials/setting-up-app.rst

```diff
@@ -1,35 +1,45 @@
-Setting up a Test-Application
+Setting up a test application
 =============================
 
 REGoth comes with lots of small test utilities, which show specific parts of the game for easier
-testing and development. For example, there are viewers for the World-Mesh, Characters, the Waynet
+testing and development.  For example, there are viewers for the world mesh, characters, the waynet
 and some others.
 
-All of those Test-Applications reside directly inside the ``src``-directory, with a name of
-``main_something.cpp``.
+All of those test applications reside directly inside the ``src`` directory, with a name of
+``main_<Subject>.cpp``.  The existing test applications shipped with REGoth are documented
+:ref:`here <execs>`.
 
+.. note::
+
+  For an overview on the existing executables and what they do exactly, see
```

Comment:
That information is already provided in the paragraph above, so the note is redundant.